<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgAdminMaterias</class>
 <widget class="QDialog" name="dlgAdminMaterias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>266</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>611</width>
    <height>266</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>611</width>
    <height>266</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Administra Materias</string>
  </property>
  <widget class="QLabel" name="lblMateria">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Materia</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leMateria">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGuardaMateria">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Guarda Materia</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNivel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nivel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbNivel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblHorasCatedra">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Horas Cátedra</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbHsCatedra">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="twMaterias">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string>Materias Cargadas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nivel</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hs Cátedra</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="lblEstado">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAlta">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alta de Materia</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leBusqueda">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblBusqueda">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Búsqueda</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leMateria</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
